src/Dofs/DofHandler.jl

```diff
@@ -12,6 +12,16 @@ Access some grid representation for the dof handler.
 """
 get_grid(dh::AbstractDofHandler)
 
+"""
+    SubDofHandler(dh::AbstractDofHandler, cellset)
+
+Create an `sdh::SubDofHandler` for the cells in `cellset`. All cells in `cellset`
+must have the same type. Fields are added to the `sdh` by using [`add!`](@ref) the 
```

Comment:
Since this is likely for aimed beginners here a maybe not so obvious question: What does "type" mean? I think we should be more specific.

Reply:
Good point, fixed!